### DISCUSSION TITLE
Tools related to SQLAlchemy - where to find them?

### DISCUSSION THREAD

**Discussion by metaperl** [sqlalchemy/sqlalchemy#9844] [Ideas]
I was looking around [the SQLAlchemy website](https://www.sqlalchemy.org) and trying to find tools that are related to it for tasks such as database design and schema migration. I did not see anything there. Is there a list of such tools somewhere?

I am about to design a small SQLite database, but do not want to do manually write the SQLAlchemy code. Nor do I want to manually upgrade my database via text should I change something about the schema, such as creating a related table or the type of a data field.

**Comment by CaselIT** [comment]
Hi,

I guess searching google or similar search engine is the best option as far as I know.
There isn't a "list of tools" page, curated or not, that's provided by the SQLAlchemy maintainers 

Regarding migration there is Alembic, that's also from SQLAlchemy maintainers, https://github.com/sqlalchemy/alembic.
Also, to generate sqlalchemy models there is https://github.com/agronholm/sqlacodegen, but I'm not sure if that's what you had in mind

**Reply by metaperl** [reply]
> There isn't a "list of tools" page, curated or not, that's provided by the SQLAlchemy maintainers

What would be the process by which a discussion could be opened about having such a page? I think it would be useful.
